x-pack/plugins/features/server/oss_features.ts

```diff
@@ -445,7 +445,7 @@ export const buildOSSFeatures = ({
             read: [],
           },
           ui: [],
-          api: ['files:manageFiles'],
+          api: ['files:manageFiles', 'files:defaultImage'],
```

Comment:
Could you please explain how the behaviour looks like for the end users?

Reply:
Good point,  I'll add this explanation to the PR description